lib/rewrite/rewrite-expr.c

```diff
@@ -58,7 +52,6 @@ log_rewrite_queue(LogPipe *s, LogMessage *msg, const LogPathOptions *path_option
       self->process(self, &msg, path_options);
     }
   msg_debug("<<<<<< rewrite rule evaluation result",
```

Comment:
As the result is not available anymore, you can delete the result word from here.